stubs/module-stub/modules/Database/Factories/ModelFactory.stub

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Modules\{{ ModuleName }}\Database\Factories;
+
+use Illuminate\Database\Eloquent\Factories\Factory;
+use Modules\{{ ModuleName }}\Models\{{ ModuleName }};
```

Comment:
This should be:
```php
 use Modules\{{ ModuleName }}\Models\{{ ResourceName }};
 ```
 
For example: A Module called "Banner" can have a Model called "Section" and a Model called "Banner" (that belongs to a Section)...